cpp/code/flight.cc

```diff
@@ -173,7 +177,41 @@ class ParquetStorageService : public arrow::flight::FlightServerBase {
   std::shared_ptr<arrow::fs::FileSystem> root_;
 };  // end ParquetStorageService
 
-arrow::Status TestPutGetDelete() {
+class HelloWorldServiceImpl : public HelloWorldService::Service {
+  grpc::Status SayHello([[maybe_unused]] grpc::ServerContext* ctx,
```

Comment:
Instead of the attribute, we could also just declare it as `grpc::ServerContext*,...`